internal/backend/runtime/omni/controllers/omni/machine_teardown.go

```diff
@@ -132,9 +141,8 @@ func (ctrl *MachineTeardownController) resetMachine(ctx context.Context, r contr
 	}
 
 	// try to wipe the machine without any attempts to retry it
-	err = client.Reset(ctx, false, false)
-	if err != nil {
-		logger.Warn("machine wipe failed", zap.Error(err))
+	if e := c.Reset(ctx, false, false); e != nil {
```

Comment:
Is using `e` instead of reassigning to `err` intentional?

Reply:
Nope! Thanks, good catch!